.travis.yml

```diff
@@ -1,10 +1,44 @@
-language: node_js
-node_js:
-  - "7"
-  - "6"
-  - "4"
+language: android
+android:
+  components:
+    # use the latest revision of Android SDK Tools
+    - platform-tools
+    - tools
+
+    # Additional components
+    - extra-google-google_play_services
+    - extra-google-m2repository
+    - extra-android-m2repository
+    - addon-google_apis-google-19
+
+    # Specify at least one system image,
+    # if you need to run emulator(s) during your tests
+    - sys-img-armeabi-v7a-android-22
```

Comment:
In my opinion `x86`(sys-img-x86-android-22) system images are faster than `armeabi`, but not really sure if travis have the support.